bcda/cli.go

```diff
@@ -113,14 +117,19 @@ func importCCLF9(filePath string) error {
 	for sc.Scan() {
 		b := sc.Bytes()
 		if len(bytes.TrimSpace(b)) > 0 {
-			fmt.Printf("\nXREF: %s\n", b[0:1])
-			fmt.Printf("Current identifier: %s\n", b[currIDStart:currIDEnd])
-			fmt.Printf("Previous identifier: %s\n", b[prevIDStart:prevIDEnd])
-			fmt.Printf("Previous identifier effective date: %s\n", b[prevIDEffDateStart:prevIDEffDateEnd])
-			fmt.Printf("Previous identifier obsolete date: %s\n", b[prevIDObsDateStart:prevIDObsDateEnd])
+
+			cclf9 := models.CCLF9{
+				CurrentNum:   string(b[currIDStart:currIDEnd]),
+				PrevNum:      string(b[prevIDStart:prevIDEnd]),
+				PrevsEfctDt:  string(b[prevIDEffDateStart:prevIDEffDateEnd]),
+				PrevsObsltDt: string(b[prevIDObsDateStart:prevIDObsDateEnd]),
+			}
+			err = db.Create(&cclf9).Error
+			if err != nil {
+				return err
+			}
```

Comment:
We'll need to clean the files up, but not sure if we want to handle that here (function side effect) or handle it via a separate call.  I wonder if @embh has thought about this for CCLF9?

Reply:
Yeah a general cleanup should happen for all the cclf files, I say a separate function to handle   the cleanup of everything.

Reply:
OK, I think we'll need a separate ticket for this.  Can you create one?

Reply:
yessir